src/Symfony/Component/Scheduler/README.md

```diff
@@ -0,0 +1,19 @@
+Scheduler Component
```

Comment:
maybe add also a default/basic changelog?

Reply:
As I'm not sure if this PR will be accepted, I'm not sure about adding a changelog as this code is not publicly available, at least for now 🤔